less/core/nav.less

```diff
@@ -24,16 +24,65 @@ html.is-nav-bottom #wrapper {
   &__inner {
     .l-container-responsive(@max-page-width);
     display: flex;
-    align-items: center;
+    align-items: stretch;
   }
 
-  // Override floats on additional `nav__btn` elements part of plugins
-  // that haven't been updated to work with data-order yet.
-  // Can be removed in time.
   &__btn {
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+
+    // Override floats on additional `nav__btn` elements part of plugins
+    // that haven't been updated to work with data-order yet.
+    // Can be removed in time.
     float: none !important;
   }
 
+  &__btn-label {
+    display: none;
+  }
+
+  &.show-label {
+    .show-nav-label(small);
+    .show-nav-label(medium);
+    .show-nav-label(large);
+  }
+
+  .dir-ltr &.has-label-auto &__btn-label,
```

Comment:
Although this selector shares the same styling as `&.has-label-right &__btn-label` should it, and the rtl varient, be defined separately?

Reply:
We do, yes. However, we usually nest the rtl style under the selector it is styling for clarity.

[button example](https://github.com/adaptlearning/adapt-contrib-core/blob/master/less/core/button.less#L21)

The rtl styles above clearly pertain to the button action selector.

The highlighted code above is good from a code optimisation perspective as it's not repeating styles but it does make the code harder to read as it's combining two unrelated selectors together `.dir-ltr &.has-label-auto &__btn-label` and `&.has-label-right &__btn-label`

Reply:
@guywillis Let me know if nesting the rtl/ltr styles is now easier to read. See [635c8c7](https://github.com/adaptlearning/adapt-contrib-core/pull/339/commits/635c8c72e329f1e5581e69b84c6738bd730ae2ff)

Reply:
It does, thank you